src/stories/Autocomplete/Component/AdvancedParameters.stories.tsx

```diff
@@ -49,3 +50,21 @@ addComponentStoryDescription(
   `const args = ${stringifyWithDefaults(TermsWithGroupSuggestions.args)}`,
   termsWithGroupSuggestionsDescription
 );
+
+export const TermsWithFilters = ComponentTemplate.bind({});
```

Comment:
Can we rename this? It makes me think Search Suggestions will be filtered but it doesn't do that. Not sure what a better name could be though 🤔